apps/src/p5lab/gamelab/GameLab.js

```diff
@@ -74,11 +73,13 @@ export default class GameLab extends P5Lab {
   }
 
   runButtonClick() {
-    // For AI Rubrics Pilot
-    const inRubricsPilot =
-      experiments.isEnabled('ai-rubrics') ||
-      experiments.isEnabled('non-ai-rubrics');
-    if (inRubricsPilot && hasScriptData('script[data-rubricdata]')) {
+    let verified;
+    if (getStore().getState().verifiedInstructor) {
+      verified = getStore().getState().verifiedInstructor.isVerified;
+    } else {
+      verified = false;
+    }
```

Comment:
A long list of integration tests were breaking trying to access the redux store - this code fixes that by setting verified to false if the verifiedInstructor store is not set.

Reply:
for next time: there is now a safe navigation operator in javascript. this can now be simplified to: 
```
const verified = !!getStore().getState().verifiedInstructor?.isVerified
```